classes/LocaleMiddleware.php

```diff
@@ -12,15 +12,19 @@ class LocaleMiddleware
      * @param  \Closure  $next
      * @return mixed
      */
-    public function handle($request, Closure $next)
-    {
+    public function handle($request, Closure $next)                                                                                                                                  
+    {                                                                                                                                                                                
         $translator = Translator::instance();
         $translator->isConfigured();
 
-        if (!$translator->loadLocaleFromRequest()) {
-            $translator->loadLocaleFromSession();
-        }
+        if( ! $translator->loadLocaleFromRequest()) {
+            if(Config::get('translate.prefixDefaultLocale')) {
```

Comment:
Please use a space after if and before condition: i.e. `if (Config`. Also, `Config` has not been loaded into this scope so please import it with a `use Config;` statement at the top of the file. This code won't work without that change. Please ensure you test this thoroughly.